packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -303,6 +310,8 @@ describe('Call Tests', () => {
       defaultServiceIndicator
     );
 
+    const mediaMetricSpy = jest.spyOn(call['metricManager'], 'submitBNRMetric');
```

Comment:
Perhaps we could call this bnrMetricSpy?

Reply:
done